test/crypto/keys/ed25519.go

```diff
@@ -23,12 +23,12 @@ var ed25519KeyPairs = []ed25519KeyPairHex{
 	{"70d92324eb8d24b7c7ed646e1996f94dcd52934a031935b9ac2d0e5bbcfa357c", "f1c41ba8a1d78f7cdc4f4ff23f3b736e30c630085697d6503e16ac899646f5ab70d92324eb8d24b7c7ed646e1996f94dcd52934a031935b9ac2d0e5bbcfa357c"},
 }
 
-type ed25519KeyPair struct {
+type Ed25519KeyPair struct {
```

Comment:
We need to make sure we only use this struct in tests so let's maybe name it explicitly with `Test`?